keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -189,6 +198,15 @@ E2E tests:
 * Add e2e tests exercising only the API operations for port ranges. Data-path 
 validation should be done by CNIs.
 
+##### Unit tests
+
+- `pkg/apis/networking/validation/validation`: `14/Jun/2022` - `92.5%`
+- `pkg/registry/networking/networkpolicy/strategy`: `14/Jun/2022` - `75.9%`
+
+##### e2e tests
+
+- test/e2e/network/netpol/network_policy_api.go: Test is optional as per the whole Network Policy suite
```

Comment:
Link to test results (as defined by the template)

Reply:
Added